### ISSUE TITLE
Custom title bar should be zoomable

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#149740]
1. VS Code mac, have custom title bar
2. Zoom in
3. Title bar is not zooming in like the rest of the workbench 🐛 

To make the custom title bar accessible it needs to be zoomable.

fyi @jrieken

**Comment by sbatten** [comment]
in general we've never done this. in fact on Windows we don't zoom in if the menubar is not present since the native title bar would not zoom. its up to the OS to decide the scale of the title bar. however, I do see a case to update the conditions if the new command center is present. what are your thoughts?

further thing to consider. we can zoom in, but zooming out (below 1 zoom level) is not an option because the window controls have a minimum size.

**Comment by isidorn** [comment]
I agree with you, and as we put more VS Code items in the title area I see a clear case to support zoom. Since the title area will feel more and more like an integral part of VS Code - and thus it should behave like the rest.

Not being able to zoom out is fine imho. I feel that is the corner case.

@kieferrm might also have thoughts, since we both noticed this bug when I was demoing the title bar search yesterday.

**Comment by bpasero** [comment]
I think we need to work with Electron to also then zoom the native window controls, otherwise it looks like this:

![image](https://user-images.githubusercontent.com/900690/171329009-fc35fe5f-c493-4071-96c3-654d5b9174cd.png)

**Comment by cpp233** [comment]
- 1.67.2 TitleBar
  ![1 67 2](https://user-images.githubusercontent.com/23007546/173187810-256d9384-333e-4fdd-b7a9-8a94e09acd72.png)

- 1.68 TitleBar
  ![1 68](https://user-images.githubusercontent.com/23007546/173187814-267a9031-348d-412e-bba3-71abceb99adb.png)

The new version is so small that it's hard to click。

**Comment by nocturnalmc** [comment]
> * 1.67.2 TitleBar
>       ![1 67 2](https://user-images.githubusercontent.com/23007546/173187810-256d9384-333e-4fdd-b7a9-8a94e09acd72.png)
> 
>     * 1.68 TitleBar
>       ![1 68](https://user-images.githubusercontent.com/23007546/173187814-267a9031-348d-412e-bba3-71abceb99adb.png)
> 
> 
> The new version is so small that it's hard to click。

Agreed. Noticed this too after updating to v1.68. I use zoom : -1 but the title bar is still big and not zoomed out like the rest of the content

**Comment by isidorn** [comment]
I have verified that the custom title bar can now we zoomable - thanks for that 🎉 
As you said zoom out is not supported which is fine, however on zoom out, the title bar does not change size but gets blurry. Like the browser anti aliasing rendering gets broken. Let me know if you would like me to file a new issue for this.

**Comment by sbatten** [comment]
@isidorn hmm yes, probably the way we implement counter zooming gives us this behavior, I don't think this is new however, nor do I think there is an easy fix for that. Would you mind verifying on stable for your machine? If it is new, I would like a separate issue to investigate.

**Comment by isidorn** [comment]
@sbatten I can reproduce on stable. So I will not file an issue. If you still need one just let me know. Thanks!
